service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -54,14 +60,30 @@ class DatasetServiceTest {
   private final AuthenticatedUserRequest user = mock(AuthenticatedUserRequest.class);
   private final DatasetId datasetId = new DatasetId(UUID.randomUUID());
   private final String sourceId = "sourceId";
+  private final String workspaceSourceId = "abc-def-workspace-id";
```

Comment:
Perhaps we could make this a dummy but valid uuid to be consistent with what Rawls expects?

Reply:
For a mocked service I don't think there's a need to make the IDs accurate, as long as we aren't depending on the ID format anywhere (which I don't think we are).